learning.yml

```
@@ -0,0 +1,126 @@
+        omero.ldap.username: cn=ldapomero,ou=users,o=dundee
+        omero.ldap.password: "{{ ldap_password }}"
+        omero.ldap.user_filter: "(objectClass=person)"
+        omero.ldap.new_user_group: Virtual Microscope
+        omero.ldap.urls: "{{ omero_server_ldap_urls }}"
+        omero.mail.config: true
+        omero.mail.from: "qa@openmicroscopy.org.uk"
+        omero.mail.host: "smtp.dundee.ac.uk"
+        omero.policy.binary_access: "+read,+write,-image,-plate"
+        omero.security.ignore_case: True
+        omero.security.keyStore: "/etc/pki/java/cacerts"
+        omero.security.trustStore: "/etc/pki/java/cacerts"
+        omero.security.keyStorePassword: "changeit"
+        omero.security.trustStorePassword: "changeit"
+        omero.sessions.timeout: 3600000
+
+    - role: ome.ldap-cert
+      ldap_host: "ldap0.dundee.ac.uk"
+
+    - role: ome.redis
+
+    - role: ome.nginx
+
+    - role: ome.nginx-configure
+      omero_web_conf: "omero/files/learning-omero-web.conf"
+
+    - role: ome.omero-web
+      ice_python_wheel: https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+      omero_web_setup_nginx: False
+      omero_web_systemd_start: False
+      omero_web_release: latest
+      omero_web_config_set:
+        omero.web.server_list:
+          - ["{{ ansible_default_ipv4.address }}", 4064, "Virtual Microscope"]
+        omero.web.prefix: '/dundee'
+        omero.web.static_url: '/dundee/static/'
+        omero.web.login_redirect:
+          redirect:
+            - webindex
+          viewname: "webindex_custom"
+        omero.web.ui.top_links:
+          - ["Virtual Microscope", "webindex", {"title": "Virtual Microscope"}]
+          - ["HELP", "https://help.openmicroscopy.org/virtual-microscope.html", {"title": "Help", "target": "new"}]
+        omero.web.ui.right_plugins:
+          - ["Acquisition", "webclient/data/includes/right_plugin.acquisition.js.html", "metadata_tab"]
+        omero.web.caches:
+          default:
+            BACKEND: django_redis.cache.RedisCache
+            LOCATION: redis://127.0.0.1:6379/0
+        omero.web.session_engine: django.contrib.sessions.backends.cache
+        omero.web.apps:
+          - "omero_gallery"
+          - "omero_iviewer"
+          - "virtualmicroscope"
+        omero.web.open_with:
+          - ["Image viewer", "webgateway", {"supported_objects": ["image"], "script_url": "webclient/javascript/ome.openwith_viewer.js"}]
+          - ["omero_iviewer", "omero_iviewer_index", {"supported_objects": ["images", "dataset", "well"], "script_url": "omero_iviewer/openwith.js", "label": "OMERO.iviewer"}]
+        omero.web.viewer.view: omero_iviewer.views.index
+        omero.web.public.enabled: true
+        omero.web.public.password: "{{ public_password }}"
+        omero.web.public.url_filter: "/(webgateway|gallery)/"
+        omero.web.public.user: public
+
+    - role: ome.omero-web-apps
```

```suggestion
    - role: ome.omero_web_apps
```

Can you add this role to `requirements.yml`?